anyway/parsers/secrets.py

```diff
@@ -1,18 +1,27 @@
 import os
+from functools import lru_cache
 import logging
 
 # Never log a secret key!
 # Note that github-secrets are not accessible for PRs from forks.
 
+SECRETS_DIR = "/run/secrets/"
+IS_SECRET_DIR = os.path.isdir(SECRETS_DIR)
```

Comment:
This is quite ugly, could this be refactored?

Reply:
The only sensible refactoring I see here is inlining `IS_SECRET_DIR` in the only place it is used, or renaming it to reveal its purpose (something like `IS_SWARM_DOCKER_SECRET`).

We can also remove the implementation entirely since it's not used or going to be used in the near future. But IMO we should use it when we have the chance, since putting secrets in environment variables is not the best strategy.

Reply:
I agree that this change has nothing to do with the PR.

Reply:
Perhaps we can remove this change from this PR, and take care of it in a designated PR.

Reply:
Removed